config/locales/de.yml

```diff
@@ -398,6 +398,7 @@ de:
       network: 'Während Ihr Code läuft, ist Port %{port} unter folgender Adresse erreichbar: <a href="%{address}" target="_blank" rel="noopener">%{address}</a>.'
       render: Anzeigen
       run: Ausführen
+      runner_in_use: Sie führen momentan Code aus. Bitte stoppen Sie die vorherige Ausführung oder warten Sie einen Moment, bevor Sie fortfahren.
```

Comment:
It seems that I cannot stop a score run (even though it is running longer than 30s). Is this expected?

Reply:
Currently, score runs cannot be stopped at all, that's correct. As of now, we are lacking a corresponding control mechanism, which clearly is a limitation. Since an exercise might contain multiple test files, the time to wait might be multiples of the permitted execution timeout: Consider an exercise with 2 test files. If each of the test files times out (and 30 seconds is set), you might need to wait a maximum of 2 (test files) x 30 seconds (timeout) = 1 minute. 

This behaviour is unchanged, but of course gets a little more annoying with blocking runners and no option to cancel the score run at all.